wd.sh

```diff
@@ -203,6 +205,28 @@ wd_add()
     fi
 }
 
+wd_addcd() {
+    local folder=$1
+    local point=$2
+    local force=$3
+    local currentdir=$PWD
```

Comment:
Do you need to quote the right hand side of the assignments? Making me worried if someone was to do `wd addcd "I'm weird"`

Reply:
The failure detection catches it
```
❯ wd addcd Im weird              
* The directory does not exist                                                   ❯ wd addcd "Im weird"
 * The directory does not exist
```
It's a good idea, but if we do that, we should do that to every function to maintain consistency

Reply:
What if it does exist though? "Im weird" is a valid folder name

Reply:
```
❯ mkdir "Im weird"
❯ wd addcd "Im weird"
 * Warp point should not contain whitespace
```